test/common.jl

```diff
@@ -0,0 +1,21 @@
+using ODE, DiffEqBase, DiffEqProblemLibrary
```

Comment:
Can you test something more here than just throws-no-error?  Maybe shape of solution? Within certain tolerance?